dashboards/dashboards.libsonnet

```
@@ -49,6 +49,73 @@ local statPanel = grafana.statPanel;
         title='Summary'
       ),
 
+    local statusMap =
+      {
+        datasource: {
+          type: 'prometheus',
+          uid: '${datasource}',
+        },
+        fieldConfig: {
+          defaults: {
+            mappings: [
+              {
+                options: {
+                  '0': {
+                    color: 'light-red',
+                    index: 1,
+                    text: 'Down',
+                  },
+                  '1': {
+                    color: 'light-green',
+                    index: 0,
+                    text: 'Up',
+                  },
+                },
+                type: 'value',
+              },
+            ],
+            links: [
+              {
+                title: '',
+                url: 'd/' + $._config.dashboardUid + 'blackbox-exporter?var-instance=${__field.labels.instance}﻿﻿﻿&var-job=${__field.labels.job}',
```

```suggestion
                url: 'd/' + $._config.dashboardUid + '/blackbox-exporter?var-instance=${__field.labels.instance}﻿﻿﻿&var-job=${__field.labels.job}',
```

A slash is missing for the link.